core/src/main/java/org/akaza/openclinica/service/crfdata/xform/EnketoAPI.java

```diff
@@ -345,7 +345,7 @@ public EnketoAccountResponse savePformRegistration() {
                 response = rest.exchange(enketoURL + "/accounts/api/v1/account", HttpMethod.PUT, request, EnketoAccountResponse.class,
                         new HashMap<String, String>());
             } catch (Exception e) {
-                System.out.println(e.getMessage());
+                logger.error(e.getMessage());
```

Comment:
you should print the entire stacktrace instead of just the message by doing: 
logger.error(e);